ansible/pmm2/post-build-actions.yml

```diff
@@ -56,24 +56,10 @@
     - name: Cleanup logs               | Cleanup logs
```

Comment:
Let's update the name of that task given that we also remove /srv/prometheus/data there

Reply:
Done!